environment-dev.yml

```diff
@@ -3,6 +3,8 @@ channels:
   - defaults
   - conda-forge
 dependencies:
+  - openmpi
```

Comment:
I think I'm OK with this, after all it's a dev install. @mloubout ?

Reply:
Oh, my bad - didn't mean to commit that part. Will remove.